tools/deploy-to-master-stable-branch.sh

```diff
@@ -40,7 +40,14 @@ fi
 echo ""
 
 echo "Building Jetpack"
-npm install -g yarn@0.16.1
+
+# Checking for yarn
+hash yarn 2>/dev/null || {
```

Comment:
At the moment, we need to make sure they're using yarn@0.16.1 specifically.  See #5635